server/request_processor.go

```diff
@@ -176,7 +183,7 @@ func (r *RpcRequest) sendTxToRelay() {
 	if err != nil {
 		r.logger.Error("[sendTxToRelay] Redis:SetTxSentToRelay failed", "error", err)
 	}
-
+	r.ethSendRawTxEntry.ShouldSendToRelay = true
```

Comment:
wrong place here, move to top of function

Reply:
updated